pom.xml

```diff
@@ -844,6 +855,11 @@
         <artifactId>spring-data-openapi</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.teiid</groupId>
+        <artifactId>spring-data-amazon-simpledb</artifactId>
+        <version>1.6.0-SNAPSHOT</version>
```

Comment:
One additional small change.  This should reference ${project.version} instead, I've updated that on master.

Reply:
Ohh yes. Idk how this got left by me. Thanks for finding and updating it.